romana-install/config.devstack.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: localhost
+  tasks:
+    - add_host: name="{{ item }}" groups="devstack_nodes,devstack_controller"
+      with_items: groups.controller
+      when: stack_type == "devstack"
+    - add_host: name="{{ item }}" groups="devstack_nodes,devstack_computes"
```

Comment:
No, it means a controller isn't "pure" because devstack uses a single local.conf for bringing a host up. The local.conf on a controller includes nova, along with all the other bits. On a compute only nova and neutron.